org.mwc.debrief.multipath2/src/org/mwc/debrief/multipath2/MultiPathView.java

```diff
@@ -91,7 +87,9 @@ public MultiPathView()
 	 */
 	public void createPartControl(final Composite parent)
 	{
-		_ui = new MultiPathUI(parent, SWT.EMBEDDED);
+	  parent.setLayout(new FillLayout());
+		_ui = new MultiPathUI(parent, SWT.EMBEDDED);  
+		// SWT.NONE allows some controls to appear - but not graph

```

Comment:
@IanMayo SWT.EMBEDDED should not be used here as we use SWT component , maybe before Chart must be used SWING component and used the SWT.EMBEDDED. I miss that it use SWT.EMBEDDED, maybe on mac/win it has no effect while it maybe having issue on linux